### DISCUSSION TITLE
Schema Browser in Db2 extension

### DISCUSSION THREAD

**Discussion by JonFP** [codefori/vscode-ibmi#1854] [Q&A]
Does anyone know how to get the names of tables etc. to show up in alpha order?  I can not work out what the order is shown in and it is making it hard for me to find the table I need. Don't want to have to set a new filter each time.

**Comment by sebjulliand** [comment]
Hi @JonFP ,
this seems to be a bug. A pull request was submitted to fix it: https://github.com/codefori/vscode-db2i/pull/198

**Reply by JonFP** [reply]
Thanks - that fix will make life a lot easier!
